### PR TITLE
Fix not-time-series model blocking OAP boots in no-init mode.

### LINKED ISSUE #9965: [Bug] OAP fails to boot up due to the absence of "ui-template"
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

OAP in non-init mode keeps complaining below errors even though the tables are created by init mode OAP's job:

```console
2022-11-16 04:54:11,985 io.kubernetes.client.informer.cache.Controller 118 [informer-controller-V1Pod] INFO  [] - informer#Controller: ready to run resync & reflector runnable
2022-11-16 04:54:11,986 io.kubernetes.client.informer.cache.Controller 127 [informer-controller-V1Pod] INFO  [] - informer#Controller: resync skipped due to 0 full resync period
2022-11-16 04:54:12,001 io.kubernetes.client.informer.cache.ReflectorRunnable 87 [controller-reflector-io.kubernetes.client.openapi.models.V1Pod-1] INFO  [] - class io.kubernetes.client.openapi.models.V1Pod#Start listing and watching...
2022-11-16 04:54:13,898 org.apache.skywalking.oap.server.core.storage.model.ModelInstaller 43 [main] INFO  [] - table: ui_template does not exist. OAP is running in 'no-init' mode, waiting... retry 3s later.
2022-11-16 04:54:16,902 org.apache.skywalking.oap.server.core.storage.model.ModelInstaller 43 [main] INFO  [] - table: ui_template does not exist. OAP is running in 'no-init' mode, waiting... retry 3s later.
2022-11-16 04:54:19,906 org.apache.skywalking.oap.server.core.storage.model.ModelInstaller 43 [main] INFO  [] - table: ui_template does not exist. OAP is running in 'no-init' mode, waiting... retry 3s later.
```

### What you expected to happen

OAP should boot up successfully.

### How to reproduce

1. Init the BanyanDB's metadata through an init-mode OAP job.
2. Start an OAP server with non-init mode.

### Anything else

Add a predicate in https://github.com/apache/skywalking/blob/master/oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java#L46 to ignore a "ui-template" model as https://github.com/apache/skywalking/blob/master/oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java#L91 did

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by hanahmily**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #9965 .
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

File: oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java [hunk 1]
```diff
@@ -44,6 +44,9 @@ public BanyanDBIndexInstaller(Client client, ModuleManager moduleManager, Banyan
 
     @Override
     public boolean isExists(Model model) throws StorageException {
+        if (!model.isTimeSeries()) { // UITemplate
+            return true;
+        }
```
Comment (wu-sheng) [inline oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java:49]: Why do we highlight this? The not-timer-series model would not be created in `#createTable`.

Comment (wu-sheng) [reply oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java:49]: OK, I just noticed https://github.com/apache/skywalking/issues/9965.

I think we need to update the title a little. The correct one as title and change log should be 

`Fix not-time-series model blocking OAP boots in no-init mode`.

```suggestion
        if (!model.isTimeSeries()) {
            return true;
        }
```

This comment is not necessary, we wouldn't remember to update this when we add next new not-time-series model.

Comment (hanahmily) [reply oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java:49]: Follow this line https://github.com/apache/skywalking/blob/master/oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java#L90

Do I have to keep it or remove them at all?

Comment (wu-sheng) [reply oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBIndexInstaller.java]: Remove this one too.

**Review by wu-sheng — APPROVED** [review]

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**